src/styles/blocks/place.scss

```diff
@@ -0,0 +1,65 @@
+.place {
+  &__photos {
+    margin-bottom: 16px;
+  }
+
+  &__photo-container {
+    position: relative;
+    width: 100%;
+    max-height: 256px;
+
+    &::after {
+      content: "";
+      display: block;
+      padding-bottom: 100%;
+
+      @include onTablet {
+        height: 321px;
+        padding-bottom: 0;
+      }
+    }
+  }
+
+  &__photo {
+    position: absolute;
+    width: 100%;
+    height: 100%;
+
+    object-fit: cover;
+    object-position: center;
+
+    transition: transform 0.5s;
+
+    &--third {
+      object-position: 80%;
+    }
+
+    &--six {
+      object-position: 80%;
+    }
+
+    &:hover {
+      z-index: 1;
+      transform: scale(1.1);
+    }
```

Comment:
here, you can try to set the scale on the img, and overflow hidden for the wrapper of this img.
then the img will increase and not cover other elements

 `
.img-wrapper:hover .img {
    transform: scale(1.2);
}
`

![Screenshot_3](https://user-images.githubusercontent.com/89162624/208906492-eb267a89-c085-40d8-a152-9b69e8398c56.png)

Reply:
Also when menu is open full page scroll works.
In the video lesson you can see how to avoid this